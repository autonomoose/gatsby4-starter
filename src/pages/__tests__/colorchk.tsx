import React from "react";
import { render } from "@testing-library/react";

import ColorPage from "../colorchk";

const mockEnqueue = jest.fn();
jest.mock('notistack', () => ({
  ...jest.requireActual('notistack'),
  useSnackbar: () => {
    return {
      enqueueSnackbar: mockEnqueue
    };
  }
}));

describe("ColorPage", () => {
  it("renders snapshot correctly", () => {
    const {container} = render(<ColorPage />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
