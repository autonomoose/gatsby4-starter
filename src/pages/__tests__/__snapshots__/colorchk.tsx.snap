// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorPage renders snapshot correctly 1`] = `
<div
  class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1ps6pg7-MuiPaper-root"
>
  <div
    class="MuiBox-root css-x3vh67"
  >
    <div
      class="MuiBox-root css-gajhq5"
    >
      <h1
        class="MuiTypography-root MuiTypography-h4 css-5lbw0b-MuiTypography-root"
      >
        Site color check h4
      </h1>
      (paper)
    </div>
    <div
      class="MuiBox-root css-12s4ewb"
    >
      <div
        class="MuiBox-root css-1s7sx0i"
      >
        site
      </div>
      <div
        class="MuiBox-root css-1n3leq2"
      >
        site2
      </div>
      <div
        class="MuiBox-root css-1ak743i"
      >
        highlight
      </div>
      <div
        class="MuiBox-root css-1a8e3qy"
      >
        highlight2
      </div>
    </div>
  </div>
</div>
`;
